src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AzureSqlFailoverGroupCmdletBase.cs

```diff
@@ -25,16 +25,6 @@ namespace Microsoft.Azure.Commands.Sql.FailoverGroup.Cmdlet
     public abstract class AzureSqlFailoverGroupCmdletBase : AzureSqlCmdletBase<IEnumerable<AzureSqlFailoverGroupModel>, AzureSqlFailoverGroupAdapter>
     {
         /// <summary>
-        /// Gets or sets the name of the server to use.
-        /// </summary>
-        [Parameter(Mandatory = true,
```

Comment:
I wanted to improve the documentation to refer to "primary" and "secondary" server rather than just server, as the distinction is important for these cmdlets.